### ISSUE TITLE
Align software docs tab order and naming

### ISSUE THREAD

**Issue by MichaIng** [MichaIng/DietPi-Docs#412]
We use a similar set of documentation tabs for software titles already, but have no consequent ordering and naming for that yet, and also the way the content is structured is different.

Let's have that aligned:
- The order should match the order in which the content usually becomes relevant, initial access/quick start and client configs first, server/services tweaks/config second, troubleshooting/logs/debugging third and updating info last, for example?
- In some cases we use half sentences for the tab names, like "Update to lastest version", in other cases single words, like "Logging" or "Directories". Let's align that as well. I personally like single word tabs, so that with an increasing number of tabs the block stays slim and not becomes many lines on small screens, but on the other hand it should be clear for inexperienced users what to find behind that tab.
- In some cases we only show a code block or file/directory path, in other cases we have some text before that. I personally think that an explaining sentence in nice to have, like "To update XY to the lastest version, run the following command from console.". Also it gives the tabbing itself more meaning. If all info is just a single command or path, then a table or list would make much more sense than tabs, which imply an additional required click.

**Comment by CouldBeThis** [comment]
I was also wondering about this and I think consistency would be great. 

FWIW I would support minimizing the use of these tabs.  They look really nice, but they are frustrating to use. The text that is in non-active tabs _cannot_ be found via ctrl-F. Locating information, especially on a long page, is made difficult. And actually I gave up a bunch of times. 

For example if you go to the site and search for `systemctl`, the first result is File Servers with 3 results. If you go to that page and ctrl-F for `systemctl` you will find nothing because all 3 results are  in non-active tabs. And in the search results it tells you what section it's in (_if_ you manage to figure out the trick to the mkdocs search) but you need to click through all the tabs to find where exactly. 

I was looking for a solution to the search part of it earlier to no avail. I wonder what the UI idea is behind these; someone must have planned for this. is it user error?

**Comment by MichaIng** [comment]
I agree the tabs bring their difficulty, but without them, the pages would grow enormously. We'd need to split per software title then, I guess.

Searching for `systemctl` is probably a bad example, as searching for any generic command without any context won't give you any reasonable result anyway, just a large list of unrelated ones, or what did you expect to find when searching for "systemctl" 😉? But indeed there are for sure important key words hidden inside the tabs as well. The most important key words should be part of the tab name.
